server/models/handlers/accountCreation.js

```diff
@@ -0,0 +1,26 @@
+/* The goal: Take in the user input from the Account Creation form in 
```

Comment:
I would recommend renaming this handler to be either `user` or `account` to allow for future flexibility and grouping of methods in handlers. Handlers should be specific enough that we can easily understand what they do, e.g., a user handler would perform include methods such as `createUser`, `removeUser`, etc. The reasoning: If a handler is created for each specific task, the codebase would undesirably increase in size, making it difficult to find desired methods.

Reply:
Hi Amy,
**Thank you for suggesting that!
My action:** I've changed the name from accountCreation to account. 
Apart form the file name, you can now see these changes the following locations:

line 3 of server/routes/publicAccountCreation.js
line 1 of server/test/accountCreation.test.js

Please confirm if this takes care of your suggestion!

Reply:
Yes, that is much better. Thank you!